src/elements/fields/SignatureField.jsx

```diff
@@ -34,7 +36,12 @@ function SignatureField({
         ref={(ref) => {
           signatureRef[servar.key] = ref;
         }}
-        onEnd={onEnd}
+        onEnd={() => {
```

Comment:
this logic should live in the onEnd function passed to this component since this is form specific info and the dashboard doesn't need it